shell.nix

```diff
@@ -4,32 +4,22 @@ pkgs.mkShell rec {
   name = "parca-agent";
```

Comment:
@maxbrunet Probably I need help to test this

Reply:
I did a quick build and it looks good